sdk/core/core-http/test/policies/tracingPolicyTests.ts

```diff
@@ -2,19 +2,8 @@
 // Licensed under the MIT License.
 
 import { assert } from "chai";
-import {
-  RequestPolicy,
-  WebResource,
-  HttpOperationResponse,
-  HttpHeaders,
-  setTracer,
-  RequestPolicyOptions,
-  TraceFlags,
-  NoOpTracer,
-  SpanOptions,
-  SpanContext,
-  NoOpSpan
-} from "../../lib/coreHttp";
```

Comment:
Might want to run `rush format` (or Prettier) on all these files because these blocks will just get expanded again once someone else does.

Reply:
These formatting changes have been happening because I have the VS Code prettier extension turned on and it will happily expand/de-expand based on line length. It doesn't seem to prefer one canonical way.

Sidebar: You've reminded me that I've been meaning to fix `rush format`, it's currently broken because @azure/core-arm doesn't have a format command.

Also when I tried to run format locally in core-http it turns out we didn't devdepend on prettier for this repo:

```
c:\src\azure-sdk-for-js\sdk\core\core-http>npm run format

> @azure/core-http@1.0.0-preview.3 format c:\src\azure-sdk-for-js\sdk\core\core-http
> prettier --write --config ../../.prettierrc.json "lib/**/*.ts" "test/**/*.ts" "*.{js,json}"

'prettier' is not recognized as an internal or external command,
operable program or batch file.
```
I'll tackle this stuff in another PR to avoid noise here. :)